frontend/src/lib/components/Animation/Animation.scss

```diff
@@ -23,4 +22,17 @@
             display: block;
         }
     }
+
+    &.Animation--large {
+        width: 100%;
+    }
+
+    &.Animation--small {
```

Comment:
Despite having just approved this I'm wondering... Is this too specific? Couldn't we better keep it as size-wise filling the view and then wrap the component in a specific size? 

Thinking the more we use the more we will have a bunch of sizes that are all randomly set for a given use-case...

Reply:
Yeah, I thought that too... The size of the container is enforced by the toastify library and Max was too little because of the space for bouncing balls...

But... that info is in my brain not the text files 🤔